apps/api/src/project/export/export.types.ts

```diff
@@ -0,0 +1,18 @@
+export enum ExportFormat {
+  JSON = 'json'
+}
+
+export interface Secret {
```

Comment:
I would rather use `Configuration` instead of separately using `Secret` and `Variable`